R/bucket_list.R

```diff
@@ -64,7 +68,7 @@ is_add_rank_list <- function(x) {
 #'   shiny::runApp(app)
 #' }
 bucket_list <- function(
-  header = NULL,
+  header = NA,
```

Comment:
I (and my other packages) have a strong preference towards using `NULL`. 

I would prefer to keep as `NULL` but will leave it to you.

Reply:
I've reverted to the original behavour (`header = NULL`) and fixed the assertion to also deal with NA values.